File: Springboot_API/RetaCantabria/src/main/java/org/example/DTO/ResenaDevueltaDTO.java
package org.example.DTO;

import org.example.Entidades.Resena;

import java.time.LocalDate;

public record ResenaDevueltaDTO (
    int idResena,
    String resena,
    LocalDate fecha,
    String nomUsuario,
    String nomRuta
    )
{
    public ResenaDevueltaDTO(Resena resena){
        this(
                resena.getIdResena(),
                resena.getResena(),
                resena.getFecha(),
                resena.getUsuario().getNombre(),
                resena.getRuta().getNombre()
                );
    }
}

File: Aplicación Escritorio/ModeloTDO/ResenaDevueltaDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ModeloDTO
{
    public class ResenaDevueltaDTO
    {
        public int idResena { get; set; }
        public String resena { get; set; }
        public DateOnly fecha { get; set; }
        public String nomUsuario { get; set; }
        public String nomRuta { get; set; }
    }
}
